### ISSUE TITLE
Autocomplete and aria-hidden (73f2c2)

### ISSUE THREAD

**Issue by cblouch** [act-rules/act-rules.github.io#993]
`autocomplete` attribute has valid value
https://act-rules.github.io/rules/73f2c2
has several test cases for “not applicable” markup. In particular is [Inapplicable Example 4](https://act-rules.github.io/rules/73f2c2#inapplicable-example-4):

```html 
<input autocomplete="username" aria-hidden="true" style="position:absolute; top:-9999em" />
```
 
This example is problematic because it violates the fourth rule of using aria by putting aria-hidden on a focusable element.
 
https://www.w3.org/TR/using-aria/#fourth
 
While it may be visually hidden via the style, it’s still in the tab order, allowing the user to move focus to “nothing”.
 
This example should either be fixed or removed.

**Comment by Jym77** [comment]
I think the point here is that while it is certainly a very bad idea (to have "invisible" focusable elements), this is not directly a violation of 1.3.5. This rule is only checking 1.3.5, no the fourth rule of ARIA.

It is a possibility to write rules whose Accessibility Requirement Mapping goes to the fourth rule of ARIA (or the others).

**Comment by jkodu** [comment]
@cblouch feel free to re-open the issue if you disagree with the above comment. Thanks.
